exercises/risiko/player.h

```diff
@@ -6,85 +6,21 @@ using namespace std;
 
 class risikoPlayer {
     private:    
-        int N,M,O;
+        int first_highest_dice, second_highest_dice, third_highest_dice;
```

Comment:
ok, però in C++ si usa il camelCase, quindi firstHighestDice, secondHighestDice e così via
i nomi sono un po' verbosi, però danno molte informazioni, prima per esempio leggendo il codice non avevo idea del loro scopo

Reply:
comunque ottimo 👍